projects.yaml

```diff
@@ -1928,3 +1935,9 @@ calculated from density functional theory as the objective function."
   license: GPL-3.0
   github_id: jeherr/element-encoder
   # package managers: none
+- name: MODNet
+  category: general-tool
+  description: "MODNet: a framework for machine learning materials properties"
+  license: MIT
+  github_id: ppdebreuck/modnet
+  homepage: https://modnet.readthedocs.io/
```

Comment:
Will comment out `homepage`. CI/CD will pick as URL for the project in the README only one of provided: homepage, docs, repo id, in that order. If the project has a repo, only pick homepage / docs over repo as project URL if the repo does not link to either of them. In most cases, the repo can be considered as root URL, since it links to homepage and docs. This is the case here.